test/browser_tests/cucumber/step_definitions/multi-select.js

```diff
@@ -15,12 +15,12 @@ defineSupportCode( function( { Then, When, Before } ) {
     multiSelect = new MultiSelect();
   } );
 
-  When( /I (.*) on the multi-select search input/,
-    async function( searchInputAction ) {
-      await browser.wait( EC.visibilityOf( multiSelect.elements.search ) );
-      await multiSelect.elements.search[searchInputAction]();
-    }
-  );
+  // When( /I (.*) on the multi-select search input/,
+  //   async function( searchInputAction ) {
+  //     await browser.wait( EC.visibilityOf( multiSelect.elements.search ) );
+  //     await multiSelect.elements.search[searchInputAction]();
+  //   }
+  // );
```

Comment:
Run `gulp lint --fix` it'll want this to be `/* */` comment.

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3706/commits/349995d72e24ecc6af9cc61915ef7ca667b0e1e4